core/trieErr.go

```diff
@@ -0,0 +1,95 @@
+package core
```

Comment:
please rename this file to getNodeFromDBErrWithKey.go

Reply:
Done.